source/dialog/request_vehicle.sqf

```diff
@@ -1,172 +1,476 @@
+          };
+    };
+
+case 22:
+    {
+          if (commandpointsblu1 >= 10) then
+          {
+           hint "Vehicle ready !";
+           commandpointsblu1 = commandpointsblu1 - 10;
+           ctrlSetText [1000, format["%1",commandpointsblu1]];
+           _vehic = "B_G_Van_01_fuel_F" createVehicle _spawnPos;
+          }
+          else
+          {
+          hint "Not enough command points";
+          };
+    };
+
+case 23:
+    {
+          if (commandpointsblu1 >= 45) then
+          {
+           hint "Vehicle ready !";
+           commandpointsblu1 = commandpointsblu1 - 45;
+           ctrlSetText [1000, format["%1",commandpointsblu1]];
+           _vehic = "B_Plane_CAS_01_F" createVehicle _spawnPos;
+          }
+          else
+          {
+          hint "Not enough command points";
+          };
+    };
+
+case 24:
+    {
+          if (commandpointsblu1 >= 40) then
+          {
+           hint "Vehicle ready !";
+           commandpointsblu1 = commandpointsblu1 - 40;
+           ctrlSetText [1000, format["%1",commandpointsblu1]];
+           _vehic = "B_MBT_01_TUSK_F" createVehicle _spawnPos;
+          }
+          else
+          {
+          hint "Not enough command points";
+          };
+    };
+
+case 25:
+    {
+          if (commandpointsblu1 >= 20) then
+          {
+           hint "Vehicle ready !";
+           commandpointsblu1 = commandpointsblu1 - 20;
+           ctrlSetText [1000, format["%1",commandpointsblu1]];
+           _vehic = "B_UGV_01_rcws_F" createVehicle _spawnPos; createVehicleCrew _vehic;
+          }
+          else
+          {
+          hint "Not enough command points";
+          };
+    };
+
+case 26:
+    {
+          if (commandpointsblu1 >= 10) then
+          {
+           hint "Vehicle ready !";
+           commandpointsblu1 = commandpointsblu1 - 10;
+           ctrlSetText [1000, format["%1",commandpointsblu1]];
+           _vehic = "B_UGV_01_F" createVehicle _spawnPos; createVehicleCrew _vehic;
+          }
+          else
+          {
+          hint "Not enough command points";
+          };
+    };
+
+case 27:
+    {
+          if (commandpointsblu1 >= 15) then
+          {
+           hint "Vehicle ready !";
+           commandpointsblu1 = commandpointsblu1 - 15;
+           ctrlSetText [1000, format["%1",commandpointsblu1]];
+           _vehic = "B_UAV_02_F" createVehicle _spawnPos; createVehicleCrew _vehic;
+          }
+          else
+          {
+          hint "Not enough command points";
+          };
+    };
+
+case 28:
+    {
+          if (commandpointsblu1 >= 15) then
+          {
+           hint "Vehicle ready !";
+           commandpointsblu1 = commandpointsblu1 - 15;
+           ctrlSetText [1000, format["%1",commandpointsblu1]];
+           _vehic = "B_UAV_02_CAS_F" createVehicle _spawnPos; createVehicleCrew _vehic;
+          }
+          else
+          {
+          hint "Not enough command points";
+          };
+    };
+
+case 29:
```

Comment:
Here's the updated patch file. http://www.maddonkeysoftware.com/DUWS-R/update-vehicle-requests-reformat.zip You should be able to just do `git apply patch update-vehicle-requests-reformat.patch` then `git add -A` followed by `git commit --amend` pressing `:wq` to close the screen where you edit the commit message.